src/routes/(admin)/dashboard/+page.svelte

```
@@ -1,11 +1,32 @@
 <script lang="ts">
+	import { goto } from '$app/navigation';
+	import { page } from '$app/state';
+
+	import StateSelection from '$lib/components/StateSelection.svelte';
+	import TagsSelection from '$lib/components/TagsSelection.svelte';
 	import Button from '$lib/components/ui/button/button.svelte';
 	import * as Dialog from '$lib/components/ui/dialog/index.js';
 	import Label from '$lib/components/ui/label/label.svelte';
```

ESLint

[error] 'Label' is defined but never used.

(@typescript-eslint/no-unused-vars)